src/test/scala/mesosphere/marathon/api/v2/ModelValidationTest.scala

```diff
@@ -74,6 +80,34 @@ class ModelValidationTest
       v.message == "Requested service port 3200 conflicts with a service port in app /app2") should be(true)
   }
 
+  test("Multiple errors within one field of a validator should be grouped into one array") {
+    val empty = ImportantTitle("")
+
+    validate(empty) match {
+      case Success => fail()
+      case f: Failure =>
+        val errors = (Json.toJson(f) \ "details").as[Seq[JsObject]]
+        errors should have size 1
+        (errors.head \ "path").as[String] should be("name")
```

Comment:
The way i got it last time is, that self appears if the error is on the top level only. Meaning if the error is in an attribute of the object, the path yields only the attribute. So self would appear as a standalone information, if the error is not in an attribute, but the object itself. I agree that prepending it in general is a coherent solution, if it does not confuse the user, I am also in for that.